src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2007,5 +2008,42 @@ private IOperation CreateUsingLocalDeclarationsOperation(BoundUsingLocalDeclarat
                                                  getChildren: () => ImmutableArray.Create<IOperation>(CreateBoundMultipleLocalDeclarationsOperation((BoundMultipleLocalDeclarations)boundNode)),
                                                  isImplicit: false);
         }
+
+        internal IPropertySubpatternOperation CreatePropertySubpattern(BoundSubpattern subpattern)
+        {
+            SyntaxNode syntax = subpattern.Syntax;
+            return new CSharpLazyPropertySubpatternOperation(this, subpattern, syntax, _semanticModel);
+        }
+
+        internal IOperation CreatePropertySubpatternMember(Symbol symbol, SyntaxNode syntax)
+        {
+            var nameSyntax = (syntax is SubpatternSyntax subpatSyntax ? subpatSyntax.NameColon?.Name : null) ?? syntax;
+            bool isImplicit = nameSyntax != syntax;
```

Comment:
>nameSyntax != syntax [](start = 30, length = 20)

Should this condition be inversed? It looks like we are lacking a test that simply asserts IOperation tree shape for this code path, consider adding one.